docstore/drivertest/drivertest.go

```diff
@@ -1485,6 +1486,11 @@ func testGetQuery(t *testing.T, _ Harness, coll *docstore.Collection) {
 			q:    coll.Query().Where("Player", "not-in", []string{"pat", "billie"}),
 			want: func(h *HighScore) bool { return h.Player != "pat" && h.Player != "billie" },
 		},
+		{
+			name: "WithGlitch",
```

Comment:
Can you also add tests for "in" and "not-in"?

Reply:
Yes it's done